README.md

```diff
@@ -79,8 +85,8 @@ gradlew.bat publishToMavenLocal -Pskip.signing
 ```java
 package hello.world;
 
-import com.goshippo.shippoSDK.Shippo;
-import com.goshippo.shippoSDK.models.operations.ListAddressesResponse;
+import com.goshippo.shippo_sdk.Shippo;
```

Comment:
This looks like a breaking / painful change to customers. I'm just thinking of the javax > jakarta changes, even when it was simple it was so annoyingly not backwards-compatible =/ our Address API has gotten customer complaints for similar casing changes, even on breaking versions

Reply:
This isn't published yet so nobody is using it yet 😄